packages/types/src/config/config.ts

```diff
@@ -295,6 +295,11 @@ export const ProsopoConfigSchema = ProsopoBasicConfigSchema.merge(
 		rateLimits: ApiPathRateLimits.default(ProviderDefaultRateLimits),
 		proxyCount: number().optional().default(0),
 		lRules: record(string(), number()).optional(),
+		authAccount: object({
+			address: string().optional(),
+			secret: string().optional(),
```

Comment:
We probably only need / want `address`.

Reply:
Sure but this is also used in the cli, where we would need the full acc to sign stuff